admin/class-boldgrid-backup-admin-cron-log.php

```diff
@@ -146,9 +197,23 @@ public function get_markup() {
 		// If we're getting the markup, the user is looking at it. Mark it as read.
 		$this->set_as_read();
 
+		$jobs = $this->get_jobs();
+
 		$log = $this->get_log();
 
-		$markup = '<h2>' . __( 'Latest Cron Notices', 'boldgrid-backup' ) . '</h2>
+		$markup = sprintf(
+			'<h2>%1$s</h2>
+			<p>%2$s %3$s</p>
+			%4$s
+			</tbody>
+			</table>',
+			__( 'Scheduled Jobs', 'boldgrid-backup' ),
+			__( 'If there are any jobs scheduled to run, they will be displayed below.', 'boldgrid-backup' ),
+			__( 'Successfully completed jobs will be removed from the list the next time the cron runs.', 'boldgrid-backup' ),
```

Comment:
Why are these not combined into 1?

Instead of `<p>%2$s %3$s</p>` just have `<p>%2$s</p>`

Reply:
Because then the line would be REALLY long. And if I break down the line inside of the __() function into two separete lines, the wordpress plugin checker throws an error for it not being a single string literal.